spec/requests/pages/questions_controller_spec.rb

```diff
@@ -123,6 +125,10 @@
         get edit_question_path(form_id: 2, page_id: 1)
       end
 
+      # after do
+      #   draft_question.delete_all
+      # end
+
```

Comment:
Think this should be deleted, uncommented, or if it needs to be kept as-is we should add another comment saying what it's for